docs/smi-trafficsplit/manifests/00-bookinfo-setup.yaml

```diff
@@ -141,7 +153,8 @@ metadata:
   name: bookinfo-gateway
 spec:
   selector:
-    istio: ingressgateway # use istio default controller
+    #istio: ingressgateway # use istio default controller
+    app: citrix-ingressgateway # use istio default controller
```

Comment:
Just curious about this change. What is the reasoning for changing this selector?

Reply:
@michelleN sorry my bad... I was testing citrix-ingressgateway with SMI Spec. Will revert it.